test/unit/carriers/fedex_test.rb

```diff
@@ -507,14 +507,25 @@ def test_tracking_info_without_status
     mock_response = xml_fixture('fedex/tracking_response_multiple_results')
     @carrier.expects(:commit).returns(mock_response)
 
-    error = assert_raises(ActiveShipping::Error) do
+    error = assert_raises(ActiveShipping::ResponseError) do
       @carrier.find_tracking_info('123456789012')
     end
 
     msg = 'Multiple matches were found. Specify a unqiue identifier: 2456987000~123456789012~FX, 2456979001~123456789012~FX, 2456979000~123456789012~FX'
     assert_equal msg, error.message
   end
 
+  def test_tracking_info_without_tracking_details
```

Comment:
You could also change it throughout this PR. Or, keep it as is and have a dedicated PR to updating this test file to `test "" do`